<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from math.iisc.ac.in/~mathbio/Net_bio.php by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 20 Jan 2017 06:55:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8" /><!-- /Added by HTTrack -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>MathBio - DST Centre for Mathematical Biology</title>
<link href="styles/style.css" rel="stylesheet" type="text/css" />
</head><body>
<table width="980" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td colspan="2" align="left" valign="middle"><a name="top" id="top"></a><img src="images/spacer.gif" width="100" height="20" alt=" " /></td>
  </tr>
  <tr>
    <td width="250" height="100" align="center" valign="middle" bgcolor="#FFFFFF"><a href="index-2.html"><img src="images/logo.jpg" alt="MathBio" width="255" height="100" border="0" /></a></td>
    <td width="730" height="100" align="left" valign="middle" bgcolor="#FFFFFF"><img src="images/banner.jpg" width="725" height="100" alt=" " /></td>
  </tr>
</table>
<table width="980" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td bgcolor="#CCCCCC"><img src="images/spacer.gif" width="100" height="5" alt=" " /></td>
  </tr>
</table>
<table width="980" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td width="980" valign="top" bgcolor="#FFFFFF" ><table width="980" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="249" valign="top" bgcolor="#FFFFFF"><br />
          <p>
<table width="200" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td><a class="left" href="index-2.html">
      <div class="leftmenucell">Home</div>
    </a></td>
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
    <td><a class="left" href="phd-programme.html">
      <div class="leftmenucell">Ph.D. Programme</div>
    </a></td>
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
    <td><a class="left" href="PhD%20Convener.html">
      <div class="leftmenucell">Convener</div>
    </a></td>
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
  <td><a class="left" href="PhD%20Participating%20Faculty.html">
      <div class="leftmenucell">Participating Faculty</div>
    </a></td>
    
  </tr>
  
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  
  <tr>
  <td><a class="left" href="Intvwprocess.html">
      <div class="leftmenucell">Interview Process</div>
    </a></td>
    
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
    <td><a class="left" href="PhD%20Students.html">
      <div class="leftmenucell">Students under Interdisciplinary Mathematical Sciences Programme</div>
    </a></td>
  </tr>
  <tr>
    <td><a class="left" href="propos.html">
      <div class="leftmenucell">Project Proposals</div>
    </a></td>
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
    <td><img src="images/spacer.gif" width="200" height="8" alt=" " /></td>
  </tr>
  <tr>
    <td><a class="left" href="contact.html">
      <div class="leftmenucell">Contact</div>
    </a></td>
  </tr>
</table>
</p></td>
        <td class="maintext" width="731" valign="top" bgcolor="#FFFFFF"><h1>&nbsp;</h1>
          <h1><strong><center>Network biology: Biomolecular structures, interactions, dynamics and implications in biological and metabolic pathways</center></strong></h1>
          <h4> <strong><center>Research group</center></strong></h4>
		  <p align="justify"><a href="http://proline.physics.iisc.ernet.in/chandralab/">Nagasuma Chandra</a> and <a href="http://pauling.mbu.iisc.ernet.in/">N. Srinivasan</a><br/>
		  <br />
		  Department of Biochemistry, IISc and Department of Biochemistry, IISc </p>
		  
          <p align="justify">	Our groups offer variety of computational and mathematical biology projects in the general areas of structure, function, dynamics and interactions of biological molecules and assemblies, metabolic and signalling pathways, pathogens and their interactions with hosts, cancer etc. A sample set of projects is given here.
</p>
<p><b>
Gaussian normal mode and evolutionary analysis to probe allostery during biomolecular associations:
</b></p>
<p>
Often when two proteins interact change in structure or dynamics is observed far away from the binding site. We would like to recognize such allosteric events starting from the 3-D structures individual proteins and their complexes with other proteins. In order to recognize such sites, we use two approaches (1) We would perform Gaussian normal mode analysis which uses an elastic network representation of protein structures and using Kirchhoffs matrix of inter-residue interactions we obtain predictions on flexible and rigid regions in the structure. Molecular dynamics studies will also be employed (2) Further, we will probe the evolutionary conservation of such putative sites to predict functional relevance.
</p>
		  <br />
		 <center> <img src="images/image1.jpg" alt="image1" width ="300" height="150" /></center><br /><br />
		  <br />
<p><b>		  
		  Recognition of determinants of a protein fold using graph network approach:
</b></p>
<p>
Many proteins of entirely different sequences adopt the same 3-D fold. The common feature that determines the same fold is not yet deciphered. In this project, we present 3-D structures of proteins of same fold as a network in which the nodes represent residues and edges represent interactions. We then employ structure-based alignment of proteins of same fold which will then be tailored with the networks of protein structures. Such an integration will allow us to recognize key determinants of protein fold formation and also aid recognition of folds starting from amino acid sequences.

	</p>	  
		  <center><img src="images/image2.jpg" alt="image2" width ="180" height="300" /></center><br /><br />
		  <br />
		  
		 <p><b> Large molecular machines formed by macromolecular assemblies
</b></p>
<p>
3-D structures of large biomolecular assemblies are deciphered increasingly by cryo electron microscopic (cryoEM) techniques. In the recent times, resolution of such structures is typically in the range of 3 to 6 Angstroms.  Therefore, fine level information is often unavailable. This project aims to formulate rules derived from the learning by analysing known high resolution structures of protein-protein complexes and using such rules in embedding atomic representation of biomolecular structures into cryoEM density maps. This process will be combined with standard steps such as flexible fitting, molecular dynamics, generation of multiple models using Gaussian normal mode analysis etc. 
</p>

<center><img src="images/image3.jpg" alt="image3" width ="400" height="500" /></center><br /><br />
		  <br />
		  <p>
	<b>	  Modelling Biochemical Pathways and  Genome-Scale Metabolic Networks 
</b></p>

<p>In a biological cell, several hundreds of biochemical reactions occur at any given point in time. Together, these reactions work as a well orchestrated whole, catering to the metabolic needs of the cell.   A number of  inter-dependencies and influences exist between different pairs of enzymes catalyzing the reactions, thus forming a complex network that has the ability to vary dynamically to suit  the state of the cell. Mathematical modeling of individual pathways and the whole network will provide an in-depth insight into the underlying molecular mechanisms and help in understanding the metabolic activity under a variety of conditions resembling both health and disease.  
</p>
<p>
The project will involve bioinformatics analysis of whole genome sequences of a selected organism, mathematical modelling including flux balance analysis, reconstruction and analyses of networks using graph theory and various other systems biology approaches.  Besides gaining fundamental insights into cellular metabolism, the work has potential applications in drug discovery and biomarker identification.
</p>

<img src="images/image4.jpg" alt="image4" width ="300" height="300" />
<img src="images/image5.jpg" alt="image5" width ="300" height="300" /><br /><br />
		  <br />
          <p align="justify">&nbsp;</p>
          <p align="justify">&nbsp;</p>
		  
          </a><br />
          </span></p></td>
      </tr>
    </table></td>
  </tr>
  <tr>
    <td height="40" valign="top" bgcolor="#FFFFFF">&nbsp;</td>
  </tr>
</table>
<table width="980" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td align="left" valign="middle" bgcolor="#FFFFFF"><img src="images/spacer.gif" width="10" height="5" alt=" " /></td>
    <td colspan="2" align="left" valign="middle" bgcolor="#FFFFFF"><hr /></td>
    <td align="left" valign="middle" bgcolor="#FFFFFF"><img src="images/spacer.gif" width="10" height="5" alt=" " /></td>
  </tr>
  <tr>
    <td width="10" align="left" valign="middle" bgcolor="#FFFFFF"><img src="images/spacer.gif" width="10" height="20" alt=" " /></td>
    <td width="480" align="left" valign="top" bgcolor="#FFFFFF"><span class="smalltext">&nbsp;Copyright: MathBio</span></td>
    <td width="480" align="left" valign="middle" bgcolor="#FFFFFF">&nbsp;</td>
    <td width="10" align="left" valign="middle" bgcolor="#FFFFFF"><img src="images/spacer.gif" width="10" height="20" alt=" " /></td>
  </tr>
  <tr>
    <td colspan="4" align="left" valign="middle" bgcolor="#990000"><img src="images/spacer.gif" width="100" height="10" alt=" " /></td>
  </tr>
  </table>
</body>

<!-- Mirrored from math.iisc.ac.in/~mathbio/Net_bio.php by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 20 Jan 2017 06:55:46 GMT -->
</html>
